### DISCUSSION TITLE
Internal Server Error after Reboot of Server

### DISCUSSION THREAD

**Discussion by martrobb** [pretix/pretix#2947] [Q&A]
Hey again!

Looking for some urgent help, our host done a whole server restart on all servers and my server was turned back on but I keep received a Internal Server Error in the pretix style/design but I didn't install it so don't know much about the SSH and docker etc. I'm good with technology but that part confuses me haha!

Any help would be appreciated, I've put an image below:
![image](https://user-images.githubusercontent.com/117538893/205625112-cf5e0ce8-2ea7-452f-81d1-31fbb1749811.png)

**Comment by raphaelm** [comment]
https://docs.pretix.eu/en/latest/admin/errors.html

**Reply by martrobb** [reply]
Yep, looked at this and it didn't help me.

**Reply by martrobb** [reply]
Here is the latest error, please dumb the meaning down haha

`ERROR 2022-11-15 18:51:22,674 django.request log Internal Server Error: /
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/base/base.py", line 219, in ensure_connection
    self.connect()
  File "/usr/local/lib/python3.9/site-packages/django/utils/asyncio.py", line 33, in inner
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/base/base.py", line 200, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/usr/local/lib/python3.9/site-packages/django/utils/asyncio.py", line 33, in inner
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/postgresql/base.py", line 187, in get_new_connection
    connection = Database.connect(**conn_params)
  File "/usr/local/lib/python3.9/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "127.0.1.1", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?`

**Reply by raphaelm** [reply]
> Yep, looked at this and it didn't help me.

Yeah, but posting the error without logs means we're unable to tell anything ;)

> Here is the latest error, please dumb the meaning down haha

It means that pretix is unable to reach your PostgreSQL instance.

If you feel you don't have the necessary Linux admin experience to manage the pretix installation but at the same time depend on its availability for business income, it might be a good idea to find someone who helps you manage it and not rely on someone replying here in their free time. We also provide commercial priority support at pretix.eu (although we still expect you to have some basic sysadmin skills), as well as a SaaS version where we take care of all the technicalities and make sure the system runs 24/7.
